packages/office-addin-debugging/src/stop.ts

```diff
@@ -5,35 +5,42 @@ import { clearDevSettings, unregisterAddIn } from "office-addin-dev-settings";
 import { readManifestFile } from "office-addin-manifest";
 import * as debugInfo from "./debugInfo";
 import { startProcess, stopProcess } from "./process";
+import * as usageDataHelper from "./usagedata-helper";
```

Comment:
we shouldn't add a file usagedata-helper in each project. This stuf can be exported from office-addin-usage-data itself. It would be better to fix the package to export things properly. That requires a major version update, e.g. a breaking change, but would be better than doing a bunch of redundant coding in each package.